js/src/forum/components/PostReactAction.js

```diff
@@ -128,15 +128,15 @@ export default class PostReactAction extends Component {
                     return [
                         <span
                             className="Button-label Button-emoji-parent"
-                            onclick={el => this.react(this.reaction ? identifier : el)}
+                            onclick={this.post.user() !== app.session.user ? (el => this.react(this.reaction ? identifier : el)) : ''}
```

Comment:
Does `onclick` on `''` not cause an error for trying to call a string? Just asking, not sure if/how this is handled.

Reply:
No error is thrown for this. I guess an alternative here would be to return `null` instead of empty string, if you'd prefer?

Reply:
Nah, if there's no error it's fine as it is.